src/lib.rs

```diff
@@ -1671,6 +1671,8 @@ impl<T> VecExt<T> for Vec<T> {
 }
 
 #[derive(Debug, Clone, Default)]
+/// Bump is a wrapper around `bumpalo::Bump` that can be cloned and also
+/// implements `Allocator`. Using this avoids lifetime polution of `Ctx`.
```

Comment:
From the PoV of API docs consumers, there should probably be something here about what this is used for...

... actually, it seems it's not exposed at all in the `run_with_ctx` signature; can we make this `pub(crate)` then?